script.js

```diff
@@ -1,4 +1,405 @@
+    }
+    return myOutputValue;
+
+    //ENTERING SCORING MODE
+  } else if (currentGameMode == scoring_mode) {
+    console.log("IT'S SCORING TIME");
+    var playerMessage = checkCurrentPlayerCards(playerDeck);
+    var computerMessage = checkCurrentPlayerCards(computerDeck);
+
+    var playertotalscore = playerDeck["totalscore"];
+    var computertotalscore = computerDeck["totalscore"];
+
+    var myOutputValue = `${playerMessage}<br><br>${computerMessage}<br><br>Player | Computer: ${playertotalscore} : ${computertotalscore}<br><br>`;
+
+    if (playerDeck["blackjack"] == true && computerDeck["blackjack"] == true) {
+      var winningMessage =
+        "BOTH BLACKJACKS?!?!?! THAT'S INCREDIBLE!!! IT'S A TIEEE";
+      playerDeck["win"] += 1;
+      computerDeck["win"] += 1;
+    } else if (
+      playerDeck["blackjack"] == true &&
+      computerDeck["blackjack"] != true
+    ) {
+      var winningMessage = "PLAYER 1 WINS BY BLACKJACK!";
+      playerDeck["win"] += 1;
+      computerDeck["loss"] += 1;
+    } else if (
+      playerDeck["blackjack"] != true &&
+      computerDeck["blackjack"] == true
+    ) {
+      var winningMessage = "COMPUTER WINS BY BLACKJACK!";
+      computerDeck["win"] += 1;
+      playerDeck["loss"] += 1;
+    } else if (
+      playertotalscore > computertotalscore &&
+      playertotalscore <= 21
+    ) {
+      var winningMessage = "PLAYER 1 WINS!";
+      playerDeck["win"] += 1;
+      computerDeck["loss"] += 1;
+    } else if (
+      computertotalscore > playertotalscore &&
+      computertotalscore <= 21
+    ) {
+      var winningMessage = "COMPUTER WINS!";
+      computerDeck["win"] += 1;
+      playerDeck["loss"] += 1;
+    } else if (playertotalscore <= 21 && computertotalscore > 21) {
+      var winningMessage = "PLAYER 1 WINS!";
+      playerDeck["win"] += 1;
+      computerDeck["loss"] += 1;
+    } else if (computertotalscore <= 21 && playertotalscore > 21) {
+      var winningMessage = "COMPUTER WINS!";
+      computerDeck["win"] += 1;
+      playerDeck["loss"] += 1;
+    } else if (playertotalscore > 21 && computertotalscore > 21) {
+      var winningMessage = "BOTH PLAYERS ARE ABOVE 21. BOTH LOST";
+      playerDeck["loss"] += 1;
+      computerDeck["loss"] += 1;
+    } else if (
+      playertotalscore <= 21 &&
+      computertotalscore <= 21 &&
+      playertotalscore == computertotalscore
+    ) {
+      var winningMessage = "IT'S A TIEEE!!!";
+    }
+    myOutputValue += winningMessage;
+  }
+  newGameMessage = "<br><br>Hit 'Submit' to start a new game!";
+  myOutputValue += newGameMessage;
+  resetGame();
   return myOutputValue;
 };
+
+/*
+===================================================================================================================
+HELPER FUNCTIONS
```

Comment:
good job on abstracting code into helper functions and using them well through your code logic